### DISCUSSION TITLE
invalid remainder

### DISCUSSION THREAD

**Discussion by hiletmis** [WalletConnect/walletconnect-monorepo#1742] [Developer Support]
Hi all, first of all congrats on new wc2. Its awesome.

I cannot perform a successful sign using etc_signTransaction method. It gives 'invalid remainder' error. All of other methods work correctly. 

Downloaded Example App to try if I was doing something wrong but it wasn't even implemented on Example App.

**Comment by llbartekll** [comment]
hey, 
we are not planning to cover this example in our sample wallet soon. Maybe take a look at other examples how to properly implement it.
but I am sure it is not a problem of our sdk.

**Reply by hiletmis** [reply]
Already tried BitKeep wallet to see if the outcome is same and BitKeep wallet also can not successfully perform the transaction.
